tests/Resources/Page/OCPDashboard/Page.robot

```diff
@@ -26,6 +26,15 @@ Maybe Click Show Default Project Button
      END
   END
 
+Create Project
+    [Documentation]     This keyword is use to create a project in OCP Console.
```

Comment:
could you remove "This Keyword etc" and write like "Create a project in OCP". Same for other docs